cc_pseudo_crawl/extract_text/requirements.txt

```diff
@@ -0,0 +1 @@
+git+git@github.com:bigscience-workshop/metadata.git
```

Comment:
Why not put this in normal `cc_pseudo_crawl/requirements.txt`?

Reply:
Because in my case, I reused my virtual environment on jz for the WG metadata and not the one you created for data_tooling.

As we had several problems with the metdata WG with different versions of datasets and I didn't want to take the risk of installing `git+git@github.com:bigscience-workshop/metadata.git` inside your virtual env in the event that it could have changed the versions of other dependencies and broke things.

So I made 2 requirements files to reflect the fact that I used a specific virtual environment for this step.